### PR TITLE
Fixed Xcode 12 Angled brackets warnings

### PR DESCRIPTION
**Pull request by ddaddy**

### CODE REVIEW

File: CoreActionSheetPicker/CoreActionSheetPicker/Pickers/ActionSheetMultipleStringPicker.h [hunk 1]
```diff
@@ -51,7 +55,7 @@ typedef void(^ActionMultipleStringCancelBlock)(ActionSheetMultipleStringPicker *
  */
 + (instancetype)showPickerWithTitle:(NSString *)title rows:(NSArray *)data initialSelection:(NSArray *)indexes target:(id)target successAction:(SEL)successAction cancelAction:(SEL)cancelActionOrNil origin:(id)origin;
 
-/// Create and display an action sheet picker with assigned target actions on success/cancel, but don't display until a subsequent call to "showActionPicker".
+/// Create and display an action sheet picker with assigned target actions on sucess/cancel, but don't display until a subsequent call to "showActionPicker".
```
Comment (noorulain17) [inline CoreActionSheetPicker/CoreActionSheetPicker/Pickers/ActionSheetMultipleStringPicker.h:58]: @ddaddy please revert this change

**Review by noorulain17 — COMMENTED** [review]
Please do the minor spelling fix

File: CoreActionSheetPicker/CoreActionSheetPicker/Pickers/ActionSheetMultipleStringPicker.h (hunk 1 above)
Comment (ddaddy) [reply CoreActionSheetPicker/CoreActionSheetPicker/Pickers/ActionSheetMultipleStringPicker.h]: How did that sneak in there?!

**Comment by ddaddy** [comment]
I've repushed with that commit omitted.

**Comment by skywinder** [comment]
Hi. Thank for the fix, @ddaddy !

tbh,  I didn't use this library for a while. So I would be appreciated if someone who uses it in prod can check this fix

@noorulain17 @KunzManuel

**Comment by noorulain17** [comment]
Closing in favor of #545
